devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -1195,7 +1195,7 @@ def reloadAllCodes() {
 		cmds << secure(zwave.userCodeV1.usersNumberGet())
 	} else {
 		sendEvent(name: "maxCodes", value: state.codes, displayed: false)
-		cmds << requestCode(1)
+		cmds << requestCode(state.checkCode)
```

Comment:
Do you know that `state.checkCode` will always be valid here? You have null checks in other places so should you use `requestCode(state.checkCode ?: 1)` here?

Reply:
@jackchi Please review the changes in PR. The changes are tested by SRI-B QA team.